data/scrapper/erplogin/erpSession.py

```diff
@@ -0,0 +1,110 @@
+import requests
+import getpass
+
+ERP_HOME_URL = 'https://erp.iitkgp.ac.in/IIT_ERP3/' #to get the session cookie
+SECURITY_QUESTION_URL = 'https://erp.iitkgp.ac.in/SSOAdministration/getSecurityQues.htm'
+LOGIN_URL = 'https://erp.iitkgp.ac.in/SSOAdministration/auth.htm'
+GET_ACAD_TOKEN_URL = 'https://erp.iitkgp.ac.in/Acad/central_breadth_tt.jsp?action=second'
+
+
+class Session:
```

Comment:
Name the class `ERPSession` and the file `session.py` instead. 😄

Reply:
Also, you can possibly subclass from `requests.Session` here.
This would provide us with a context manager, thus one could use this in `with` statements.